app/imports/components/Users/AdminPanel/UsersManagement/UsersList/index.js

```diff
@@ -0,0 +1,17 @@
+import { Meteor } from 'meteor/meteor';
+import { composeWithTracker } from 'react-komposer';
+import UsersList from './usersList.jsx';
+
+const composer = (props, onData) => {
+    const handler = Meteor.subscribe('userData');
+
+    if (handler.ready()) {
+        const users = Meteor.users.find({});
+        onData(null, {
+            users,
+        });
+    }
+    return 0;
```

Comment:
Remove this `return 0;`, function returns `undefined` by default and it is sufficient